packages/tools/devtools/devtools-browser-extension/package.json

```diff
@@ -68,9 +68,14 @@
 		"@fluentui/react": "^8.109.4",
 		"@fluid-experimental/devtools-core": "workspace:~",
 		"@fluid-experimental/devtools-view": "workspace:~",
+		"@fluidframework/common-definitions": "^0.20.1",
 		"@fluidframework/common-utils": "^1.1.1",
+		"@fluidframework/core-interfaces": "workspace:~",
+		"@microsoft/1ds-core-js": "^3.2.13",
+		"@microsoft/1ds-post-js": "^3.2.13",
 		"react": "^17.0.1",
-		"react-dom": "^17.0.1"
+		"react-dom": "^17.0.1",
+		"tslib": "^1.10.0"
```

Comment:
Is this a peer of one of the 1ds libraries or something?

Reply:
Yep.